src/app/util/attribute-storage.cpp

```diff
@@ -1245,7 +1245,7 @@ void emAfLoadAttributeDefaults(EndpointId endpoint, bool ignoreStorage, Optional
                 ptr                                 = nullptr; // Will get set to the value to write, as needed.
 
                 // First check for a persisted value.
-                if (!ignoreStorage && am->IsAutomaticallyPersisted())
+                if (am->IsAutomaticallyPersisted())
                 {
                     VerifyOrDie(attrStorage && "Attribute persistence needs a persistence provider");
```

Comment:
Not part of your PR, but...what is this? Is this just a weird way to work around not having a msg? Seems like VerifyOrDieWithMsg would make more sense. I mean, don't block the PR on this, but if you have to run another round of CI, it seems like it would make sense to update.